remoteappmanager/db/interfaces.py

```diff
@@ -0,0 +1,81 @@
+from abc import ABCMeta, abstractmethod
+
+from traitlets import HasTraits, Unicode, Bool
+
+
+class ApplicationConfig(HasTraits):
```

Comment:
I do not think that there is any advantage in having these as HasTraits class. I would expect this to but just a simple able class.